api/modules/PlacesAPI/views.py

```diff
@@ -0,0 +1,47 @@
+import requests
+import requests_cache
+from datetime import timedelta
+
+from rest_framework import status
+from rest_framework.decorators import api_view
+from rest_framework.response import Response
+
+from api.modules.PlacesAPI.response import PlaceResponse
+from api.modules.PlacesAPI.constants import PLACES_API_SEARCH
+
+hour_difference = timedelta(days=1)
+requests_cache.install_cache(expire_after=hour_difference)
+
+
+@api_view(['GET'])
+def get_places(request, latitude, longitude, place_keyword):
+    """
+        Return list of places matched
+        :param request:
+        :param latitude:
+        :param: longitude:
+        :param : place_keyword:
+        :return: 503 if Places api fails
+        :return: 200 successful
+    """
+    try:
+        api_response = requests.get(
+            PLACES_API_SEARCH.format(latitude=latitude, longitude=longitude, place_keyword=place_keyword)
+        )
+        api_response_json = api_response.json()
+        if api_response.status_code == 503:
+            error_message = "Service Unavailable"
+            return Response(error_message, status=status.HTTP_503_SERVICE_UNAVAILABLE)
+        response = []
+        suggestions = api_response_json['results']
+        for place in suggestions:
+            result = PlaceResponse(
+                title=place['title'],
+                href=place['href']
+            )
+            result_as_json = result.to_json()
+            response.append(result_as_json)
+    except Exception as e:
```

Comment:
Have different exceptions for 
+ incorrect / expired API keys
+ throttling error (if possible)